tritium-jmh/src/jmh/java/com/palantir/tritium/microbenchmarks/ProxyBenchmark.java

```diff
@@ -49,6 +53,11 @@
 @State(Scope.Benchmark)
 @SuppressWarnings("designforextension")
 public class ProxyBenchmark {
+    static {
+        System.setProperty("org.slf4j.simpleLogger.log.performance", LoggingLevel.TRACE.name());
+        TestLogs.setLevel("performance", LoggingLevel.TRACE.name());
+        TestLogs.logTo("/dev/null");
+    }
```

Comment:
I don't think we're benchmarking what we want. I see both logback-classic and slf4j-simple on my classpath for this module, slf4j binds to logback, so performance loggers are not enabled at the trace level.

Reply:
We may want to consider benchmarking using log4j2 over logback/slf4j-simple since the latter two tend to be allocation heavy and may overshadow improvements in the tritium implementation.

Reply:
yeah, we need to setup and test logging as it would be in normal production environment with log4j2 or logback.

Reply:
removing the `sl4j-simple` dependency from `tritium-test` for now, will do more thorough logger cleanup and benchmark tests in a separate PR